templates/workloads/services/lb-web/cf.yml

```diff
@@ -321,7 +321,7 @@ Resources:
 {{- if .Aliases }}
         - Field: 'host-header'
           HostHeaderConfig:
-            Values: {{ .Aliases }}
+            Values: {{ fmtSlice .Aliases }}
```

Comment:
We seem to follow the YAML syntax when it comes to string slice in other CFN templates:
```yaml
Values:
  - value1
  - value2
```
Perhaps we should follow the same pattern here?

Reply:
Good question i just think it would be less error-prone for future format changes when we format a string of lists.